### ISSUE TITLE
Unhandled 'error' event derails project generation

### ISSUE THREAD

**Issue by akfranco** [banderson/generator-redux#32]
Right after the Install Dependencies prompt whether 'yes' or 'no' is selected:

```
events.js:141
      throw er; // Unhandled 'error' event
      ^

TypeError: this.getPackageVersions is not a function
    at deps (/Users/kamuela/.nvm/versions/node/v4.2.2/lib/node_modules/generator-redux/generators/app/index.js:119:12)
    at /Users/kamuela/.nvm/versions/node/v4.2.2/lib/node_modules/generator-redux/node_modules/yeoman-generator/lib/base.js:429:16
    at processImmediate [as _immediateCallback] (timers.js:383:17)
```

**Comment by sugarshin** [comment]
Same error with Node v4.2.3

**Comment by banderson** [comment]
@KamuelaFranco this should now be fixed in the latest published version (0.3.3) due to #33 being merged. Let me know if you are still experiencing problems!
